test/fixtures/wpt/html/webappapis/structured-clone/structured-clone-battery-of-tests-harness.js

```diff
@@ -0,0 +1,44 @@
+/**
+ * Runs a collection of tests that determine if an API implements structured clone
+ * correctly.
+ *
+ * The `runner` parameter has the following properties:
+ * - `setup()`: An optional function run once before testing starts
+ * - `teardown()`: An option function run once after all tests are done
+ * - `preTest()`: An optional, async function run before a test
+ * - `postTest()`: An optional, async function run after a test is done
+ * - `structuredClone(obj, transferList)`: Required function that somehow
+ *                                         structurally clones an object.
```

Comment:
Just curious: why is this called with `await` in all of the tests? `structuredClone` doesn't return a promise, does it?